### PR TITLE
cloudhub enhancement

### PR DESCRIPTION
**Pull request by spambot000**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Enhance access control of CloudHub API

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
1. Some edge nodes with older version certificates may not be authenticated by cloudcore if this feature is turned on.
```

### CODE REVIEW

File: staging/src/github.com/kubeedge/viaduct/pkg/client/quic.go [hunk 1]
```diff
@@ -4,14 +4,13 @@ import (
...
 
-	"github.com/lucas-clemente/quic-go"
-	"k8s.io/klog/v2"
-
 	"github.com/kubeedge/beehive/pkg/core/model"
 	"github.com/kubeedge/viaduct/pkg/api"
 	"github.com/kubeedge/viaduct/pkg/comm"
 	"github.com/kubeedge/viaduct/pkg/conn"
 	"github.com/kubeedge/viaduct/pkg/lane"
+	"github.com/lucas-clemente/quic-go"
```
Comment (Shelley-BaoYue) [inline staging/src/github.com/kubeedge/viaduct/pkg/client/quic.go:12]: Sort import by three-stages format

**Review by Shelley-BaoYue — COMMENTED** [review]
If you need to update vendor, you should update go.mod file and then execute hack/update-vendor.sh and update-vendor-license.sh.

**Comment by spambot000** [comment]
@Shelley-BaoYue I've updated the package `authorization` and added some test cases.

File: staging/src/github.com/kubeedge/viaduct/pkg/client/quic.go [hunk 2]
```diff
@@ -4,8 +4,8 @@ import (
 	"fmt"
 	"sync"
 
-	"github.com/lucas-clemente/quic-go"
-	"k8s.io/klog/v2"
+	quic "github.com/lucas-clemente/quic-go"
+	klog "k8s.io/klog/v2"
```
Comment (Shelley-BaoYue) [inline staging/src/github.com/kubeedge/viaduct/pkg/client/quic.go:8]: "github.com/lucas-clemente/quic-go"
"k8s.io/klog/v2"
is enough. same with other places, but I don't think it's going to affect anything. : )

**Review by Shelley-BaoYue — COMMENTED** [review]
Overall it looks good. : ） 
Please rebase newest master branch

File: staging/src/github.com/kubeedge/viaduct/pkg/client/quic.go (hunk 2 above)
Comment (spambot000) [reply staging/src/github.com/kubeedge/viaduct/pkg/client/quic.go]: Fixed. Please review again.

**Review by fisherxu — COMMENTED** [review]
/approve
Overall looks great!
/assign @WillardHu @Shelley-BaoYue

**Review by Shelley-BaoYue — COMMENTED** [review]
/lgtm

**Pull request merged**